WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsDetailsViewModel.swift

```diff
@@ -421,10 +421,15 @@ private extension SiteStatsDetailsViewModel {
     }
 
     func countriesRows() -> [StatsTotalRowData] {
-        return periodStore.getAllCountries()?.map { StatsTotalRowData.init(name: $0.label,
-                                                                           data: $0.value.displayString(),
-                                                                           countryIconURL: $0.iconURL,
-                                                                           statSection: .periodCountries) }
+        return periodStore.getAllCountries()?.map {
+            // This hack is here just temporarily until #11342 gets merged which exposes the "code" property nicely.
```

Comment:
The same question I posed for `SiteStatsPeriodViewModel.swift` applies here as well.